Sources/LocalizedStringKit/LocalizedStringKit.m

```diff
@@ -153,4 +153,8 @@ + (NSBundle *)getLocalizedStringKitBundle:(NSString *_Nullable)bundleName
   return nil;
 }
 
+void SetPrimaryBundleName(NSString *_Nonnull bundleName) {
```

Comment:
We probably want to prefix this with "LSK" as well: `LSKSetPrimaryBundleName()`

Reply:
updated